DOCUMENTATION.md

```diff
@@ -357,6 +357,26 @@ Parameters : User user
 bool result = await OpenFoodAPIClient.login(user);
 ```
 
+#### Register a new user
+
+This functions creates a new Open Food Facts user. 
+
+```dart
+Parameters: User user
+    
+Status status = await OpenFoodAPIClient.register(user);
+```
+
+Although `user.name` and `user.email` are nullable they must not be null for registering.
```

Comment:
Why are they nullable then? 😄

Reply:
Thanks for your review Florian-Schoenherr.
For most applications of the user, i.e. for editing products etc., only the `user_id` and the `password` are used. I have now changed it so that the optional parameters are no longer specified via the user object but as individual functions parameters.

Reply:
seems like a nicer API :)
hope I can help some more, even if only a little

Reply:
Every help is apricieated 💯